### PR TITLE
docs: fixup global members

### PR DESCRIPTION
**Pull request by gkatsev**
There are still two globals that I can't figure out how to fix properly:
* TextTrackCue typedef in the TextTrackCueList
* mergeOptions

Though, I don't think those two should hold this PR up.

### CODE REVIEW

**Comment by BrandonOCasey** [comment]
`TextTrackCue` should probably be `TextTrackCueList~TextTrackCue` to keep it in that classes scope? I could be wrong on that though. LGTM otherwise

**Comment by gkatsev** [comment]
I tried using that but let me try again.

**Comment by BrandonOCasey** [comment]
If that change worked I approve this PR

**Comment by gkatsev** [comment]
Yeah, I think so. It seems like it sometimes just gets cached or something so sometimes just refreshing the page doesn't seem to help. The change definitely doesn't hurt, though.
